conda-environment.yml

```diff
@@ -23,3 +23,4 @@ dependencies:
   #- pympler
   #- matplotlib-base
   #- sqlparse
+  #- h5py
```

Comment:
I think we install the tutorial dependencies by default in the conda-forge feedstock, so this might need to move to the main section, not sure.

But oddly, I don't see `requests` here?

Reply:
```suggestion
  #- h5py
  #- requests
```